packages/nitro-protocol/contracts/interfaces/IForceMove.sol

```diff
@@ -140,10 +139,8 @@ interface IForceMove {
      */
     event ChallengeRegistered(
         bytes32 indexed channelId,
-        // everything needed to respond or checkpoint
         uint48 turnNumRecord,
         uint48 finalizesAt,
-        address challenger,
```

Comment:
If the goal is to save gas, the vast majority of the data on this event can be removed. I think only the `channelId` needs to be emitted; we can then lean on the "get data from `calldata`" technique to fill out the rest.

Reply:
https://github.com/statechannels/statechannels/issues/3699